src/main/java/emissary/place/KffHashPlace.java

```diff
@@ -22,7 +22,7 @@ public class KffHashPlace extends ServiceProviderPlace {
 
     public static final String SKIP_KFF_HASH = "SKIP_KFF_HASH";
 
-    private boolean useSbc = true;
+    private boolean useSbc = false;
```

Comment:
Won't that get overwritten by line 54 if the entry is not in the config?

Reply:
Addressed